src/ORM/Connect/DBSchemaManager.php

```diff
@@ -28,6 +30,16 @@ abstract class DBSchemaManager
      */
     private static $check_and_repair_on_build = true;
 
+    /**
+     * For large databases you can declare a list of DataObject classes which will be excluded from
+     * CHECK TABLE and REPAIR TABLE queries during dev/build. Note that the entire inheritance chain
+     * for that class will be excluded.
+     *
+     * Only use this configuration if you know what you are doing and have identified specific models
+     * as being problems during your dev/build process.
+     */
+    private static array $exclude_dataobjects_for_checks = [];
```

Comment:
As the title suggests, we want to exclude models.. or DO models.. in the end, they are classes.
```suggestion
    private static array $exclude_classes_from_db_checks = [];
```
or
```suggestion
    private static array $exclude_models_from_db_checks = [];
```
or similar

Reply:
I'll go with `exclude_models_from_db_checks` as it keeps the context that it's DataObject subclasses specifically.

Reply:
Done